mautic/mautic
# Fix locked category

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | NA
| Related developer documentation PR URL | NA
| Issue(s) addressed                     | 



#### Description:
No one can modify categories created by another user (not even admin account). It seems like a bug with the permissions in the project.
But in fact it is because the category is still "under edition".

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create category, save
3. Quit edition without "closing"
4. Ask another user to edit it > browser prompt error
